recipes/docly/meta.yaml

```diff
@@ -1,61 +0,0 @@
-{% set name = "docly" %}
```

Comment:
Please don't modify any other recipes in here, just leave those files alone.

Reply:
That was my bad. I had accidentally added that recipe earlier but reverted. This is that commit.

Reply:
That's showing up as a change vs. master. If it was already merged, I'm not sure what to do other than rebase and eliminate your problematic commits.